libs/ui/src/lib/build-status/build-status.component.scss

```diff
@@ -0,0 +1,49 @@
+@import '~@angular/material/theming';
+@include mat-core();
+
+$hint-palette: mat-palette($mat-blue-gray, 800);
+$failure-palette: mat-palette($mat-red);
+$success-palette: mat-palette($mat-green);
+
+.container {
+  height: 100%;
+
+  .mat-progress-spinner {
+    display: inline-block;
+    margin: 0 4px;
+  }
+
+  h3 mat-icon {
+    margin: 0 4px;
+  }
+}
+
+.inner-container {
+  height: calc(100% - 5px);
```

Comment:
Why 5px? Mind extracting the value to a variable with a descriptive name?

Reply:
Not needed, good catch! Probably something left in there when I was playing around with styling.